scripts/_pkg-sources.R

```diff
@@ -1,6 +1,6 @@
 list(
   bslib = "rstudio/bslib@main",
-  shiny = "rstudio/shiny@main",
+  shiny = "rstudio/shiny@feat/textAreaAutoResize",
```

Comment:
Once https://github.com/rstudio/shiny/pull/4210 is merged, revert this change, and re-build dependencies

```suggestion
  shiny = "rstudio/shiny@main",
```